### PR TITLE
Plasma draining OB

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

It drains plasma for a bit but is otherwise harmless to marines
WP buffed against benos as a side effect

## Why It's Good For The Game

More push options

## Changelog
:cl:
add: Plasma draining OB 
balance: WP's plasma drain effect is much stronger
/:cl:

### CODE REVIEW

**Comment by InterroLouis** [comment]
I don't like that fuel is being taken away with the shells being added.

**Comment by TiviPlus** [comment]
The fuel amount is the same, it was just moved onto the other racks

**Comment by InterroLouis** [comment]
You remove 4 fuel from 1 rack and add a total of 3 fuel to other racks on the Pillar of Spring, and remove 4 fuel from 1 rack and add a total of 2 fuel to other racks on the Theseus. On the Sulaco you just add 3 shells without adding any fuel. The fuel amount is not the same.

**Comment by TiviPlus** [comment]
PoS:
remove 4 fuel from one rack
Add 1 fuel to 3 racks
add 1 fuel
Theseus:
Add 1 fuel to 3 racks
add 1 fuel to 1 rack

File: code/game/objects/structures/orbital_cannon.dm [hunk 1]
```diff
@@ -25,9 +25,9 @@
 
 	if(!GLOB.marine_main_ship.ob_type_fuel_requirements)
 		GLOB.marine_main_ship.ob_type_fuel_requirements = list()
-		var/list/L = list(4,5,6)
+		var/list/L = list(3,4,5,6)
 		var/amt
-		for(var/i=1 to 3)
+		for(var/i=1 to 4)
```
Comment (Rohesie) [inline code/game/objects/structures/orbital_cannon.dm:30]:
```suggestion
		for(var/i in 1 to 4)
```

**Pull request merged**